.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+name: CI
```

Comment:
it requires a tikv and pd server to be deployed to perform the test, how about renaming it to `Integration Test` and run it for every pull request?

Reply:
The integration test is at L29. `CI` contains both check format and integration test.